src/services/gateway.service.ts

```diff
@@ -202,8 +186,35 @@ function getThemeLogo(
   }
 }
 
+function uploadThemeLogo(
```

Comment:
shouldn't we call `uploadAppLogo`, `removeAppLogo`?

cc: @ajithranka

Reply:
Idea behind naming it ThemeLogo instead of AppLogo was we have different logos for different themes and orientation

Themes: `light` and `dark`
Orientation: `horizontal` and `vertical`